concepticondata/conceptlists/BirketSmith-1928-510.tsv

```diff
@@ -0,0 +1,511 @@
+BirketSmith-1928-510-369	369	Woman’s sister’s husband			Relationship
+BirketSmith-1928-510-370	370	Husband’s brother			Relationship
+BirketSmith-1928-510-371	371	Wife’s younger sister’s husband			Relationship
+BirketSmith-1928-510-372	372	Husband’s younger brother’s wife			Relationship
+BirketSmith-1928-510-373	373	Wife’s elder sister’s husband			Relationship
+BirketSmith-1928-510-374	374	Husband’s elder brother’s wife			Relationship
+BirketSmith-1928-510-375	375	Son’s father-in-law			Relationship
+BirketSmith-1928-510-376	376	Daughter’s father-in-law			Relationship
+BirketSmith-1928-510-377	377	Son’s mother-in-law			Relationship
+BirketSmith-1928-510-378	378	Daughter’s mother-in-law			Relationship
+BirketSmith-1928-510-379	379	Adopted father			Relationship
+BirketSmith-1928-510-380	380	Adopted mother			Relationship
+BirketSmith-1928-510-381	381	Adopted son			Relationship
+BirketSmith-1928-510-382	382	Adopted daughter			Relationship
+BirketSmith-1928-510-383	383	I	1209	I	Pronouns
+BirketSmith-1928-510-384	384	You (sing.)	1215	THOU	Pronouns
+BirketSmith-1928-510-385	385	He, she it	262	HE OR SHE OR IT	Pronouns
+BirketSmith-1928-510-386	386	We (dual.)	2634	WE TWO	Pronouns
+BirketSmith-1928-510-387	387	We (plur.)	1212	WE	Pronouns
+BirketSmith-1928-510-388	388	You (dual.)	2111	YOU TWO	Pronouns
+BirketSmith-1928-510-389	389	You (plur.)	1213	YOU	Pronouns
+BirketSmith-1928-510-390	390	They (dual. & plur.)	817	THEY	Pronouns
+BirketSmith-1928-510-391	391	This	1214	THIS	Pronouns
+BirketSmith-1928-510-392	392	These (dual. & plur.)			Pronouns
+BirketSmith-1928-510-393	393	Who	1235	WHO	Pronouns
+BirketSmith-1928-510-394	394	What	1236	WHAT	Pronouns
+BirketSmith-1928-510-395	395	One	1493	ONE	Numerals
+BirketSmith-1928-510-396	396	Two	1498	TWO	Numerals
+BirketSmith-1928-510-397	397	Three	492	THREE	Numerals
+BirketSmith-1928-510-398	398	Four	1500	FOUR	Numerals
+BirketSmith-1928-510-399	399	Five	493	FIVE	Numerals
+BirketSmith-1928-510-400	400	Six	1703	SIX	Numerals
+BirketSmith-1928-510-401	401	Seven	1704	SEVEN	Numerals
+BirketSmith-1928-510-402	402	Eight	1705	EIGHT	Numerals
+BirketSmith-1928-510-403	403	Nine	1483	NINE	Numerals
+BirketSmith-1928-510-404	404	Ten	1515	TEN	Numerals
+BirketSmith-1928-510-405	405	Eleven	1706	ELEVEN	Numerals
+BirketSmith-1928-510-406	406	Twelve	1707	TWELVE	Numerals
+BirketSmith-1928-510-407	407	Thirteen	1708	THIRTEEN	Numerals
+BirketSmith-1928-510-408	408	Fourteen	2451	FOURTEEN	Numerals
+BirketSmith-1928-510-409	409	Fifteen	1709	FIFTEEN	Numerals
+BirketSmith-1928-510-410	410	Sixteen	2454	SIXTEEN	Numerals
+BirketSmith-1928-510-411	411	Seventeen	2453	SEVENTEEN	Numerals
+BirketSmith-1928-510-412	412	Eighteen	1713	EIGHTEEN	Numerals
+BirketSmith-1928-510-413	413	Nineteen	1714	NINETEEN	Numerals
+BirketSmith-1928-510-414	414	Twenty	1710	TWENTY	Numerals
+BirketSmith-1928-510-415	415	Many	1198	MANY	Numerals
+BirketSmith-1928-510-416	416	How	1239	HOW	Particles
+BirketSmith-1928-510-417	417	Last, at	1529	LAST (FINAL)	Particles
```

Comment:
I'm not sure what the author was going for here, but wouldn't "at last" rather mean something like "finally"?

Reply:
I'll double check with the lexemes.

Reply:
If it's a particle as indicated by the category, then better to unmap it, I think.